src/novmm/loader/linux_setup.go

```diff
@@ -69,75 +71,75 @@ static inline void set_header(
 import "C"
 
 import (
-    "novmm/machine"
-    "novmm/platform"
-    "unsafe"
+	"novmm/machine"
+	"novmm/platform"
+	"unsafe"
 )
 
 func SetupLinuxBootParams(
-    model *machine.Model,
```

Comment:
Looks like you reformatted the whole file with ``gofmt`` inserting tabs instead of four spaces for indent.